webpack.config.js

```diff
@@ -106,17 +106,14 @@ const browserConfig = (/*env, argv*/) => {
   };
 
   //const patterns = [{ from: "package.json", to: "_data/package.json" }];
-  let patterns = [{
-    from: "src/assets/favicons",
-    to: "site/favicons",
-  }];
+  let patterns = [];
```

Comment:
Makes sense to move these to their actual location at this point 👍 

This is however resulting in a build error:

```
ValidationError: Invalid options object. Copy Plugin has been initialized using an options object that does not match the API schema.
 - options.patterns should be a non-empty array.
 ```

Reply:
Yep, somehow that was missed, now fixed.